packages/react-native-worklets/Common/cpp/worklets/SharedItems/Shareables.cpp

```diff
@@ -438,4 +447,25 @@ jsi::Value ShareableScalar::toJSValue(jsi::Runtime &) {
   }
 }
 
+ShareableTurboModuleLike::ShareableTurboModuleLike(
+    jsi::Runtime &rt,
+    const jsi::Object &object,
+    const std::shared_ptr<jsi::HostObject> &proto)
+    : Shareable(TurboModuleLikeType) {
+  proto_ = std::make_unique<ShareableHostObject>(rt, proto);
+  properties_ = std::make_unique<ShareableObject>(rt, object);
```

Comment:
Let's move these to the initializer list:
```suggestion
    : Shareable(TurboModuleLikeType),
  proto_(std::make_unique<ShareableHostObject>(rt, proto)),
  properties_(std::make_unique<ShareableObject>(rt, object))
```